src/TreeTransform.jl

```diff
@@ -324,7 +319,6 @@ function fixed_point(ctx::RewriteContext, node::T, rebuild::Bool) where { T }
         if rewritten2 === rewritten
             ctx.fixed_points[node] = rewritten
             ctx.fixed_points[rewritten] = rewritten
-            # println("  rewritten to ($(objectid(rewritten))) $rewritten")
             return rewritten
         end
```

Comment:
not for this PR, but it might be good to create custom exception types for this package so they are more easily handle-able by other code bases